source/settings.txt

```diff
@@ -49,14 +48,11 @@ Tasks
 Learn more 
 ----------
 
-To learn how to configure your |compass| settings through the |compass-short| 
-interface, see :ref:`compass-interface-settings`.
+- :ref:`compass-interface-settings`.
```

Comment:
[suggestion]  Lists that don't consist of complete sentences don't need ending periods!

Reply:
Ah thanks for catching that! Looks like it got left from when I deleted the sentence.